.wiki/_DV/Laws/StandardOperatingProcedure.txt

```diff
@@ -420,3 +420,30 @@ In order for a medical evaluation to be considered lawfully applicable, the CMO
 
 ==Quarantine==
 Medical staff are afforded the right to quarantine individuals that currently suffer from infectious or contagious diseases, and may confine them within pre-designated quarantine areas in the course of treatment, until the point that they are no longer infectious or harmful to the rest of the crew.
+=Engineering Regulations=
+===Wiring===
+All wiring must be concealed from public for their safety.
+In non public areas, catwalk is recommended to be placed for safety and mobility of workers.
+
+Wiring is to be held straight and tidy, without unnecessary looping, to avoid clumping, short circuiting and trpping hazard for the crew.
+==Atmospheric Regulations==
+===Habitable Rooms===
+Rooms with purpose of supporting organic life require the following:
+* Air pressure around 1 atmosphere (101.325 kPa)
+* Air temperature between 15° and 35° Celsius.
+* Barrier made of 2 meters (3.28 ft) of reinforced glass or stronger from non habitable space.
+* Air mixture supporting majority of station's crew.
+* Presence of hazardous gasses below 1 mol.
+
+===Pressure In Pipe Networks===
+It is recommended to set taget pressure in any pipe network double than one atmosphere (202.65 kPa).
```

Comment:
this is a meme that would gimp distro for no reason, distro pump != the nitrogen and oxygen pumps and mixer
additionally assuming this is specifically for distro, theres no harm at all in having max pressure distro: unanchoring pressure increase is negligible and it just makes distro last longer during a spacing preventing it failing and having stationwide pressure loss

this would also imply you need to have waste at   200kpa and not 0?!

tritium and frezon setups also exist